libraries/AC_CustomControl/AC_CustomControl_PID.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AC_PID/AC_PID.h>
+#include <AC_PID/AC_P.h>
+
+#include "AC_CustomControl_Backend.h"
+
+#ifndef CUSTOMCONTROL_PID_ENABLED
+    #define CUSTOMCONTROL_PID_ENABLED AP_CUSTOMCONTROL_ENABLED
+#endif
+
+#if CUSTOMCONTROL_PID_ENABLED
+
+// default rate controller PID gains
+#define AC_ATTITUDE_CONTROL_ANGLE_P                     4.5f             // default angle P gain for roll, pitch and yaw
+
+#ifndef AC_ATC_MULTI_RATE_RP_P
+  # define AC_ATC_MULTI_RATE_RP_P           0.135f
+#endif
+#ifndef AC_ATC_MULTI_RATE_RP_I
+  # define AC_ATC_MULTI_RATE_RP_I           0.135f
+#endif
+#ifndef AC_ATC_MULTI_RATE_RP_D
+  # define AC_ATC_MULTI_RATE_RP_D           0.0036f
+#endif
+#ifndef AC_ATC_MULTI_RATE_RP_IMAX
+ # define AC_ATC_MULTI_RATE_RP_IMAX         0.5f
+#endif
+#ifndef AC_ATC_MULTI_RATE_RP_FILT_HZ
+ # define AC_ATC_MULTI_RATE_RP_FILT_HZ      20.0f
+#endif
+#ifndef AC_ATC_MULTI_RATE_YAW_P
+ # define AC_ATC_MULTI_RATE_YAW_P           0.180f
+#endif
+#ifndef AC_ATC_MULTI_RATE_YAW_I
+ # define AC_ATC_MULTI_RATE_YAW_I           0.018f
+#endif
+#ifndef AC_ATC_MULTI_RATE_YAW_D
+ # define AC_ATC_MULTI_RATE_YAW_D           0.0f
+#endif
+#ifndef AC_ATC_MULTI_RATE_YAW_IMAX
+ # define AC_ATC_MULTI_RATE_YAW_IMAX        0.5f
+#endif
+#ifndef AC_ATC_MULTI_RATE_YAW_FILT_HZ
+ # define AC_ATC_MULTI_RATE_YAW_FILT_HZ     2.5f
+#endif
```

Comment:
I would be tempted to just hard code these defines in the PID constructions, its not like anyone would need to change these, its only a example. They are exact duplicated of the defines in AC_Attitude control which could lead to confusion. You may even just be able to include those directly?

Reply:
We were already including AC_AttitudeControl_Multi through frontend, so I removed them.